charts/enterprise-metrics/templates/graphite-querier-schemas-configmap.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.graphite.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "enterprise-metrics.fullname" . }}-graphite-schemas
+  labels:
+    app: {{ template "enterprise-metrics.name" . }}
+    chart: {{ template "enterprise-metrics.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  storage-schemas.conf: |
+    [default]
+    pattern = {{ .Values.graphite_querier.schemasConfiguration.storageSchemas.pattern }}
```

Comment:
sorry for asking for more changes again. but i think that to make this properly configurable it should support a dynamic number of ini sections and not only `default`. So this would need to be a loop which iterates over the ini sections and creates one in each iteration.

Reply:
WDYT about this instead https://github.com/grafana/helm-charts/pull/872/commits/ff92a95d8903dac573b8e739c3bb933950826fc1

Rather than going through the complexity of multiple sections and their attributes, we let the user create a variable with the raw inputs and thas what gets idented into the configmap.

Reply:
That works, thanks :+1: